### PR TITLE
Use X-Forwarded-For when websockets are proxied

### PR DESCRIPTION
**Pull request by kalda341**
This fixes the number of websockets for an IP issue when HaProxy is in front of Kamailio

### CODE REVIEW

File: kamailio/websockets-role.cfg [hunk 1]
```diff
@@ -62,65 +62,83 @@ route[NAT_WEBSOCKETS_CORRECT]
...
 }
 
+route[WEBSOCKETS_ORIGIN_IP]
+{
+    if ($hdr(X-Forwarded-For) == $null) {
+        xlog("L_INFO", "websocket|log|request from has no X-Forwarded-For, originates from $si\n");
+        $var(orig_ip) = $si;
+    } else if ($si =~ "MY_WEBSOCKET_PROXY_IP_ADDRESS") {
+        xlog("L_INFO", "websocket|log|request X-Forwarded-For $hdr(X-Forwarded-For) from $si\n");
+        $var(orig_ip) = $hdr(X-Forwarded-For);
```
Comment (lazedo) [inline kamailio/websockets-role.cfg]: whatif `X-Forwarded-For` is sent but `MY_WEBSOCKET_PROXY_IP_ADDRESS` matches ?

Comment (kalda341) [reply kamailio/websockets-role.cfg]: @lazedo I'm not sure how to change the value at runtime. Can you explain what I'm looking for in the presence-role?

Comment (sergey-safarov) [reply kamailio/websockets-role.cfg]: Think need to use ACL for this.
This because HA proxy may be also have redurdancy like in Amazon.

**Comment by lazedo** [comment]
@kalda341 hey, i incorporated your changes in #92

**Comment by lazedo** [comment]
@kalda341 Hi, please check latest master as i believe your changes were integrated

**Pull request closed without merging**